Source/Scene/Expression.js

```diff
@@ -123,6 +126,44 @@ define([
 
     function log2(number) {
         return CesiumMath.logBase(number, 2.0);
+
+    }
+
+    function distance(x, y) {
+        if (typeof x === 'number' && typeof y === 'number') {
+            return Math.abs(x - y);
+        } else if (x instanceof Cartesian2 && y instanceof Cartesian2) {
+            return Cartesian2.distance(x ,y);
+        } else if (x instanceof Cartesian3 && y instanceof Cartesian3) {
+            return Cartesian3.distance(x, y);
+        } else if (x instanceof Cartesian4 && y instanceof Cartesian4) {
+            return Cartesian4.distance(x, y);
+        } else {
+            throw new DeveloperError('Function distance requires matching types of inputs. Arguments are ' + x + ' and ' + y + '.');
```

Comment:
Wrap each of these three `DeveloperError` with 

```
//>>includeStart('debug', pragmas.debug);
throw new DeveloperError('.....')
//>>includeEnd('debug');
```

so that it is not included in release builds.